riff-raff/app/deployment/actors/TasksRunner.scala

```diff
@@ -69,5 +69,5 @@ class TasksRunner(stopFlagAgent: Agent[Map[UUID, String]]) extends Actor with Lo
 
 object TasksRunner {
   trait Message
-  case class RunDeployment(uuid: UUID, deployment: DeploymentTasks, rootReporter: DeployReporter, queueTime: DateTime) extends Message
+  case class RunDeployment(uuid: UUID, deployment: DeploymentTasks, rootResources: DeploymentResources, queueTime: DateTime) extends Message
```

Comment:
`rootResources` is a bit of a misnomer here. It makes sense for a reporter to be a rootReporter, less so for a resource. A reporter has a context attached which determines where in the log tree any reports are appended. This ensures that logs for a task are all bundled together.